packages/asc-ui/package.json

```diff
@@ -25,13 +25,13 @@
     "@amsterdam/asc-assets": "^0.25.2",
     "classnames": "^2.2.6",
     "deepmerge": "^4.2.2",
-    "polished": "^4.0.1",
+    "polished": "^4.0.3",
     "react-uid": "^2.3.0"
   },
   "peerDependencies": {
-    "react": ">=16.8.6",
-    "react-dom": ">=16.8.6",
-    "styled-components": ">=4.4.1"
```

Comment:
Wouldn't it be better to leave these at `16`? The code base doesn't depend on version `17` specifically, or does it?

Reply:
ah good one, yes I didn't expect `ncu` would also update peerDeps